packages/cli/openapi-ir-to-fern/src/buildEndpoint.ts

```diff
@@ -405,11 +404,20 @@ function getRequest({
                             {
                                 type: getTypeFromTypeReference(propertyTypeReference),
                                 docs: getDocsFromTypeReference(propertyTypeReference),
-                                name: property.nameOverride
+                                name: property.nameOverride,
+                                availability
                             }
                         ];
                     }
-                    return [property.key, propertyTypeReference];
+                    return [
+                        property.key,
+                        availability
+                            ? {
+                                  type: getTypeFromTypeReference(propertyTypeReference),
+                                  availability
+                              }
```

Comment:
we should share this code with lines 404-408 since it looks like we may already be dropping things like `docs` so its error prone